src/commands/add-readme.ts

```diff
@@ -15,21 +15,21 @@ async function action(args: { options: { path: string; }; }) {
 
   if (args.options.path) {
     path = argPath.startsWith("~") ? 
-      HOME + argPath.slice(1) :
-      argPath;
+      HOME + argPath.slice(1)
+      : argPath;
```

Comment:
unless you were asked to change this, i would keep the : on the previous row, as it makes it clear that this piece of code continues on the next line
```suggestion
      HOME + argPath.slice(1) :
      argPath;
```